.gitignore

```diff
@@ -48,3 +48,7 @@ jspm_packages
 # Lockfiles
 yarn.lock
 package-lock.json
+
+# Cypress
+cypress/videos/
+cypress/screenshots/
```

Comment:
Might be cleaner if this is moved to a file in `cypress/.gitignore`:

```
/videos
/screenshots
```

Reply:
Great point! :grinning: Moved in 349c0ed2d